integrations/ivy/src/app/store-ivy.module.ts

```diff
@@ -0,0 +1,27 @@
+import { NgModule } from '@angular/core';
+import { NgxsModule } from '@ngxs/store';
+import { NgxsReduxDevtoolsPluginModule } from '@ngxs/devtools-plugin';
+import { NgxsFormPluginModule } from '@ngxs/form-plugin';
+import { NgxsLoggerPluginModule } from '@ngxs/logger-plugin';
+import { NgxsStoragePluginModule } from '@ngxs/storage-plugin';
+import { NgxsWebsocketPluginModule } from '@ngxs/websocket-plugin';
+
+import { AppState } from './app.state';
+
+// @noinspection - disable checking duplicate code in integrations files for IDEA
```

Comment:
What are these comments for?

Reply:
If you use Jetbrains IDEA, then the development environment swears at repeated files in integration tests

Reply:
done